winml/dll/winml.rc

```diff
@@ -0,0 +1,42 @@
+// This file REQUIRES the following external definitions:
```

Comment:
Are you going to do similar metadata changes for onnxruntime.dll as well or is that going to be a separate PR? cc Ori Levari

Reply:
Yeah, ill take them on today at some point and out a separate PR into master.